src/app/store.ts

```diff
@@ -1,10 +1,17 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { todosSlice } from '../features/todos';
+import { filterSlice } from '../features/filter';
+import { currentTodoSlice } from '../features/currentTodo';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices({
+  todos: todosSlice.reducer,
+  filter: filterSlice.reducer,
+  currentTodo: currentTodoSlice.reducer,
+});
```

Comment:
Ensure you are using `combineReducers` here instead of `combineSlices` to properly combine the reducers for the Redux store.